tests/Unit/SettingsPage/ValidationsTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Parse.ly settings page validations tests.
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\SettingsPage\Validations;
+
+use Parsely\Parsely;
+use Yoast\WPTestUtils\BrainMonkey\TestCase;
+use Parsely\UI\Settings_Page;
+
+
+/**
+ * Parse.ly settings page validations tests.
+ */
+final class SettingsValidationsTest extends TestCase {
+	/**
+	 * Internal Parsely variable.
+	 *
+	 * @var Parsely $parsely Holds the Parsely object.
+	 */
+	private static $parsely;
+
+	/**
+	 * Internal Settings_Page variable.
+	 *
+	 * @var Settings_Page $settings_page Holds the Settings_Page object.
+	 */
+	private static $settings_page;
+
+	/**
+	 * Runs the setup process before each test.
+	 */
+	public function set_up(): void {
+		parent::set_up();
+
+		self::$parsely       = new Parsely();
+		self::$settings_page = new Settings_Page( self::$parsely );
```

Comment:
Instantiating a class in the `UI` namespace, to test logic around a string, seems like too much coupling going on. 

Seems like perhaps the validation of the string should be handled by a validation-specific class, outside of the `UI` namespace, which the class in the `UI` namespace can then have injected, and defer to.

Reply:
I've put a mention of this comment in the above issues. Marking this conversation as resolved, as I think that changing this is part of a bigger chunk of work outside of this PR's scope.